README.md

```diff
@@ -56,6 +56,29 @@ composer require --dev friendsofphp/php-cs-fixer
 For more details and other installation methods, see
 [installation instructions](./doc/installation.rst).
 
+### Run with Docker
+
+You can take a ready built docker image to run ``php-cs-fixer``.
+
+```console
+docker run -v $(pwd):/code ghcr.io/php-cs-fixer/php-cs-fixer:$FIXER_VERSION fix src
+```
+
+Or integrate as check into Gitlab CI like this:
+
+```yaml
+php-cs-fixer:
+  image: ghcr.io/php-cs-fixer/php-cs-fixer:$FIXER_VERSION
+  script:
+    php-cs-fixer fix --diff --dry-run --format=txt src
+```
```

Comment:
I would like to keep readme minimalistic, docker usage info is great, but integrating with single specific CI, I would not put it in readme at all.

Drop it, move to https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/installation.rst or create specific doc file for CI integration (then I would even prioritise github over gitlab, but OK to have both, or none)

Reply:
Again - can we improve it in consequent PR?

Reply:
I see no value in externalising all the comments. If we have typo in PR, you would just fix it in PR, not externalise to new PR, right? :)